composer.json

```diff
@@ -24,7 +24,8 @@
   "autoload": {
     "psr-4": {
       "Terminus\\": ["php/", "php/Terminus/"],
-      "Pantheon\\Terminus\\": "src/"
+      "Pantheon\\Terminus\\": "src/",
+      "Pantheon\\Terminus\\Tests\\": "tests/"
```

Comment:
Reading further, it seems you only added this for `tests/CommandTestCase.php`.  It should not be necessary to add an entry to the autoloader for this file (which I **do** think belongs immediately inside `tests`), as phpunit should already be loading it.

Finally, if you **did** want to add some `tests` directory to the autoloader, you should put it in `autoload-dev` rather than `autoload`.